src/backend/distributed/commands/rename.c

```diff
@@ -63,14 +64,30 @@ PreprocessRenameStmt(Node *node, const char *renameCommand,
 		return NIL;
 	}
 
-	/* check whether we are dealing with a sequence here */
-	if (get_rel_relkind(objectRelationId) == RELKIND_SEQUENCE)
+	/*
```

Comment:
I think we need some more comments here. It would not be clear to the readers that you can do `ALTER TABLE ` on a view. We should mention that (even though we discussed in detail, I was confused while reading the code). At least expand as much as the comment in `PreprocessAlterTableStmt `?